app/src/main/java/com/parishod/watomatic/adapter/SupportedAppsAdapter.kt

```diff
@@ -0,0 +1,62 @@
+package com.parishod.watomatic.adapter
+
+import android.content.pm.PackageManager
+import android.graphics.drawable.Drawable
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.parishod.watomatic.R
+import com.parishod.watomatic.model.App
+import com.parishod.watomatic.model.preferences.PreferencesManager
+import com.parishod.watomatic.model.utils.Constants
+import kotlinx.android.synthetic.main.supported_apps_list.view.*
+
+
+class SupportedAppsAdapter(private val listType: Constants.EnabledAppsDisplayType, private var supportedAppsList: ArrayList<App>): RecyclerView.Adapter<SupportedAppsAdapter.AppsViewHolder>() {
```

Comment:
Any reason to use `ArrayList` over `List`?

Reply:
no particular reason

Reply:
Changed it to `List` dd34942